tools/dm/cluster-operations.md

```diff
@@ -71,7 +71,7 @@ The information maintained by DM-master includes the following two major types,
 
 When DM-master is restarted, it automatically requests the task information from each DM-worker instance and rebuilds the corresponding relationship between the task and DM-worker. However, at this time, DM-worker does not resend the sharding DDL information, so it might occur that the sharding DDL lock synchronization cannot be finished automatically because of the lost lock information.
 
-To resolve this issue, follow the steps described in [Troubleshooting Sharding DDL Locks](/tools/dm/troubleshooting-sharding-ddl-locks.md#condition-three-dm-master-restarts).
+To resolve this issue, follow the steps described in [Handle Sharding DDL Locks Manually](/tools/dm/manually-handling-sharding-ddl-locks.md#scenario-3-some-dm-workers-are-temporarily-unreachable-during-the-ddl-unlocking-process).
```

Comment:
Now, when DM-master is restarted, it will rebuild the relationship and also re-fetch the DDL information from DM-worker, so the DDL lock can be resolved automatically. 

I forgot to update the Chinese document, this is my fault.

this section can be explained in Chinese:

```
DM-master 中维护的信息主要包括以下两类，且在重启时不会进行持久化。

- task 与 DM-worker 的对应关系
- sharding DDL lock 相关信息

当 DM-master 重启时，会自动从各 DM-worker 请求 task 信息并重建 task 与 DM-worker 的对应关系，且会重新从各 DM-worker 处获取 sharding DDL 信息。因此能正确地重建对应的 DDL lock 并自动地完成 sharding DDL lock 同步。
```